qhub/schema.py

```diff
@@ -382,6 +382,15 @@ class QHubExtension(Base):
     envs: typing.Optional[typing.List[QHubExtensionEnv]]
 
 
+class IngressTerraformOverrides(Base):
+    load_balancer_annotations: typing.Optional[typing.Dict]
+    load_balancer_ip: str
+
+
+class Ingress(Base):
+    terraform_overrides: IngressTerraformOverrides
```

Comment:
I'd prefer if this is `terraform_overrides: Any` and we defer to terraform for the checks.

Reply:
So no need for the `IngressTerraformOverrides`

Reply:
The problem with this is, "Any" won't reach terraform, as we parse these in the `input_vars.py`, so anything other than this won't be passed, unless we pass them explicitly.

Reply:
Ah, I see your comment below now.